experimental/grpc_attestation/src/attestation.rs

```diff
@@ -174,7 +181,11 @@ impl AttestationServer {
 }
 
 #[tonic::async_trait]
-impl RemoteAttestation for AttestationServer {
+impl<F, S> RemoteAttestation for AttestationServer<F>
+where
+    F: 'static + Send + Sync + Clone + FnOnce(Vec<u8>) -> S,
+    S: std::future::Future<Output = anyhow::Result<Vec<u8>>> + Send + Sync + 'static,
```

Comment:
In `impl<F, S> AttestationServer<F>` you didn't have `'static`.
Should this template specification be the same everywhere?

Reply:
`'static` is needed because of the lifetime bounds on `AttestationServer`. This is a requirement from `tonic`, because here we are implementing `RemoteAttestation` for `AttestationServer`. But `RequestHandler` does not have this lifetime bound. 
I think it is okey that the template specifications are different.